assets/sass/_docs.scss

```diff
@@ -0,0 +1,14 @@
+// CSS overrides for generated SDK and CLI docs.
+
+.section-docs {
+
+    .section {
```

Comment:
Does this class only show up in the generated Python docs? If so, should these styles go in `_api-python.scss`?

Reply:
The fix is aimed at an issue that shows up in multiple places (not just Python) &mdash; but now I see there are other areas I've missed here, so will need to augment this.